app/renderer/components/reduxComponent.js

```diff
@@ -45,8 +46,16 @@ class ReduxComponent extends ImmutableComponent {
   }
 
   shouldComponentUpdate (nextProps, nextState) {
-    return Object.keys(nextState).some((prop) => nextState[prop] !== this.state[prop]) ||
-      Object.keys(nextProps).some((prop) => nextProps[prop] !== this.props[prop])
+    return Object.keys(nextState).some((prop) => {
+      return isList(nextState[prop])
```

Comment:
since this is a little more complicated now it would probably be better to pass an instance method to both instead of using anonymous functions. It would be slightly faster anyway and avoid the overhead of creating and GCing the closure